README.md

```diff
@@ -1,2 +1,12 @@
 # inji-vc-renderer
-Description: Kotlin and JS library to render VC with SVG template support
+This repository contains Kotlin and JS library to render VC with SVG template support.
+
+### Kotlin
+- Kotlin directory has the library which generates aar and jar for VC Rendering.
+- It also has the example application which uses the library code.
+- More information on the  [Kotlin library Read Me](kotlin/Readme.md)
+
+### Js
+- Js directory has the javscript library for VC Rendering.
+- It also has the example application which uses the library code to use the exported modules from the library.
+- More information on the  [Js library Read Me](kotlin/Readme.md)
```

Comment:
More information on the Js library [check here](js/Readme.md)

Reply:
Done